collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala

```diff
@@ -79,6 +81,28 @@ object CollectionRollerMain extends LazyLogging {
     val (solr, solrService, collectionRoller) = createServices(parsedArgs)
     try {
       collectionRoller.collectionList().foreach(println)
+
+    } finally {
+      solrService.close()
+      solr.shutdown()
+    }
+    logger.info("Ending Collection Roller List App")
+  }
+
+  private def listApplicationsVerbose(parsedArgs: CollectionRollerCliArgsParser): Unit = {
+    logger.info("Starting Collection Roller List App")
+
+    val (solr, solrService, collectionRoller) = createServices(parsedArgs)
+    try {
+
+      println("Applications:")
+      collectionRoller.collectionList().foreach(println)
+      println("Aliases/Collections:")
+
+      for ((k, v) <- collection.mutable.LinkedHashMap(
```

Comment:
I think this output format might be more readable:

```
appname
    alias_latest
        coll7
    alias_all
        coll1
        coll3
        coll6
```

what do you think?

Reply:
This looks good. I will list it that way.